pyproject.toml

```diff
@@ -1,14 +1,65 @@
-[tool.black]
-line-length = 120
-# By default, `black` will ignore skip configuration when paths are explicitly provided.
-# In order for `pre-commit` to respect this configuration, `force-exclude` needs to be explicitly set.
-force-exclude = 'migrations'
-
-[tool.isort]
-# https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#isort
-profile = "black"
-# By default, `isort` will ignore skip configuration when paths are explicitly provided.
-# In order for `pre-commit` to respect this configuration, `filter_files` needs to be set to true.
-# https://jugmac00.github.io/blog/isort-and-pre-commit-a-friendship-with-obstacles/
-filter_files = true
-skip_glob = ["*/migrations/*", "config/*"]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".bzr",
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".git-rewrite",
+    ".hg",
+    ".ipynb_checkpoints",
+    ".mypy_cache",
+    ".nox",
+    ".pants.d",
+    ".pyenv",
+    ".pytest_cache",
+    ".pytype",
+    ".ruff_cache",
+    ".svn",
+    ".tox",
+    ".venv",
+    ".vscode",
+    "__pypackages__",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    "site-packages",
+    "venv",
+]
+
+# Same as Black.
+line-length = 88
+indent-width = 4
+
+# Assume Python 3.8
+target-version = "py38"
```

Comment:
Having the latest updates in mind, I think this should be:

```suggestion
# Assume Python 3.12
target-version = "py312"
```

From [the `ruff` docs](https://docs.astral.sh/ruff/settings/#target-version), it seems it is supported.